admin_guide/iptables.adoc

```diff
@@ -11,18 +11,76 @@
 
+All components independently work with iptables without knowledge of how other components
+are using them. This makes it very easy for one component to break another component's configuration.
+Further, {product-title} and Docker assume that iptables remains set up exactly as they have set it up.
+They may not detect changes introduced by other components and if they do there maybe some lag in
+implementing the fix. In particular, {product-title} does monitor and fix problems, however, Docker does not.
 
-== Restarting
-Docker doesn't monitor the iptables rules that it adds for exposing ports from containers and hence if
-iptables service is restarted, then these rules are lost. So, to safely restart iptables, it is
-recommended that the rules are saved and restored. 
+IMPORTANT: You must make sure that whatever changes you make to the iptables configuration on a node
+do not impact the operation of {product-title} and Docker. Also, changes will often need to be made on
+all nodes in the cluster. Be careful, the setup is very complex and it is very easy to break {product-title}
+and Docker networking.
+
+The chains, order of the chains and rules in the kernel iptables must be properly set up on each node
+in the cluster for {product-title} and Docker networking to work properly. There are several tools and
+services that are commonly used in the system that interact with the kernel iptables and can
+accidentally impact {product-title} and Docker.
+
+=== iptables
+Iptables is a low level tool that can be used to set up, maintain, and inspect the tables of IPv4
+packet filter rules in the Linux kernel.
+
+WARNING: This is a very powerful tool that can manage all aspects of the kernel iptables. It assumes
+you know what you are doing.
+
+Independent of other use, such as a firewall, {product-title} and Docker manage chains in some of the tables.
+The chains are inserted in specific order and the rules are specific to their needs.
+
+IMPORTANT: iptables --flush [chain] can remove key required configuration. Be very careful when using this.
+
+When making changes consider limiting your changes to the OS_FIREWALL_ALLOW chain in the filter table.
+
+=== iptables.service
+The iptables service supports a local network firewall. It assumes total control of the iptables
+configuration and when it starts it flushes and restores the complete iptables configuration.
+The restored rules are from its configuration file (/etc/sysconfig/iptables).
+The config file is not kept up to date during operation so every restart the dynamically added rules are lost.
+
+WARNING: Stopping and starting the iptables.service will destroy configuration that is required by
+{product-title} and Docker. {product-title} and Docker are not notified of the change.
+
+WARNING: Use of iptables.service is not recommended in conjunction with {product-title} or Docker.
+
+Iptables.service can be disabled and masked to prevent accidental execution:
```

Comment:
The install part of the doc explicitly installs it. The intention is to move to firewalld when ansible is ready. so there will be more churn here and hopefully we can eventually just delete the discussion on iptables.service.